project/Whitesource.scala

```diff
@@ -2,15 +2,17 @@
  * Copyright (C) Lightbend Inc. <https://www.lightbend.com>
  */
 
-package build.play.grpc
+import sys.process.Process
 
 import sbt._
 import sbt.Keys._
 import sbtwhitesource.WhiteSourcePlugin.autoImport._
 import sbtwhitesource._
-import com.typesafe.sbt.SbtGit.GitKeys._
```

Comment:
Does this mean we can remove sbt-git from project/plugins.sbt? I like that because it breaks `git worktree` (at least by default)